### DISCUSSION TITLE
kubevela fluxcd addon version  v:0.33.0 ..not have bearerToken for git access

### DISCUSSION THREAD

**Discussion by workman-Lu** [fluxcd/flux2#3823] [Q&A]
it is difficult that have no this verify type for us

**Comment by kingdonb** [comment]
It isn't clear what this issue is about, Kubevela is a third-party project and it isn't maintained by the Flux team.

If the kubevela maintainers are open to feedback, maybe you can elaborate on a report for them, and link it back here if you need our help? (I don't think any Flux maintainers use kubevela now, so I'm not sure how much help we'll be able to provide on this topic.)

**Reply by kingdonb** [reply]
It looks like the kubevela flux addon lives here:

https://github.com/kubevela/catalog - maybe you can create an issue here @workman-Lu ?

**Reply by kingdonb** [reply]
Flux 0.39 added the `bearerToken` support for the first time. If KubeVela fluxcd addon is versioned according to Flux versions, then 0.33.0 is a very old version of Flux. (Too old for the `bearerToken` feature.)

It might just need to be updated. There have been some breaking changes in Flux, I'm not sure how complicated it would be to update.
